test/format/wts.c

```diff
@@ -196,6 +196,23 @@ configure_timing_stress(char **p, size_t max)
     CONFIG_APPEND(*p, "]");
 }
 
+/*
+ * configure_file_manager --
+ *     Configure file manager settings.
+ */
+static void
+configure_file_manager(char **p, size_t max)
+{
+    CONFIG_APPEND(*p, ",file_manager=[");
+    if (GV(FILE_MANAGER_CLOSE_HANDLE_MINIMUM))
```

Comment:
Since these are integer types, let's do `GV(FILE_MANAGER_CLOSE_HANDLE_MINIMUM) != 0`, to be explicit that we are dealing with integer values.